pkg/sentry/kernel/fd_table_unsafe.go

```diff
@@ -162,14 +165,6 @@ func (f *FDTable) setAll(ctx context.Context, fd int32, file *fs.File, fileVFS2
 		}
 	}
 
-	// Adjust used.
-	switch {
-	case orig == nil && desc != nil:
-		atomic.AddInt32(&f.used, 1)
-	case orig != nil && desc == nil:
-		atomic.AddInt32(&f.used, -1)
-	}
```

Comment:
Why don't we need this anymore?

Reply:
As f.used only used for count the total number of used fds in fd_table. `f.fdBitmap.GetNumOnes()` can achieve same goals.